### PR TITLE
Make sure the browser does not show the access-key in the history

### PR DESCRIPTION
**Pull request by NicolasDorier**
Reopening it!

### CODE REVIEW

**Comment by ShahanaFarooqui** [comment]
Should i merge and publish it now?

**Comment by ShahanaFarooqui** [comment]
Tested on my local, not working. Going offline for the night.

**Comment by NicolasDorier** [comment]
almost there :)

**Comment by NicolasDorier** [comment]
@ShahanaFarooqui where is the source code of the front end?

There is one thing missing: You need to include the `access-key` cookie in you request to `api/authenticate`

File: controllers/authenticate.js [hunk 1]
```diff
@@ -6,10 +6,30 @@ var upperCase = require('upper-case');
...
 exports.authenticateUser = (req, res, next) => {
-  password = atob(req.body.password);
   if(+common.rtl_sso) {
-    if (common.cookie === password) {
+    const access_key = req.cookies['access-key'];
+    res.clearCookie("access-key");
+    if (common.cookie === access_key) {
       const token = jwt.sign(
         { user: 'Custom_User', lndConfigPath: common.lnd_config_path, macaroonPath: common.macaroon_path },
         'default_secret_key'
```
Comment (NicolasDorier) [inline controllers/authenticate.js]: maybe we can use `common.cookieParserSecret` instead?

**Comment by NicolasDorier** [comment]
also, the cookie is only for HTTPS!

**Comment by NicolasDorier** [comment]
trying it

**Pull request merged**